jmxtrans-output/jmxtrans-output-kafka/src/main/java/com/googlecode/jmxtrans/model/output/kafka/KafkaWriter.java

```diff
@@ -77,7 +77,7 @@ public KafkaWriter(
 		kafkaProperties.setProperty("zk.connect", Settings.getStringSetting(settings, "zk.connect", null));
 		kafkaProperties.setProperty("serializer.class", Settings.getStringSetting(settings, "serializer.class", null));
 		this.producer= new Producer<String,String>(new ProducerConfig(kafkaProperties));
-		this.topics = asList(topics.split(","));
+		this.topics = asList(Settings.getStringSetting(settings, "topics", null).split(","));
```

Comment:
It seems that here we use a default value for topic of `null`, which means that we will throw a null pointer exception when doing the split. We should have a better validation here. Not really sure how kafka works, if we can have a safe default (empty list?) or not. If we can have a safe default, we should use it, else, we need to throw an exception at construction.